### PR TITLE
don't let writers change team avatar

### PR DESCRIPTION
**Pull request by zanderz**
The service was letting team writers change the avatar and description - that was the root cause, right?

For a generic error, the description will be shown, too:
image

### CODE REVIEW

File: go/teams/service_helper.go [hunk 1]
```diff
@@ -1613,7 +1613,7 @@ func CanUserPerform(ctx context.Context, g *libkb.GlobalContext, teamname string
 
 	ret.DeleteChannel = admin
 	ret.RenameChannel = writer
-	ret.EditChannelDescription = writer
+	ret.EditChannelDescription = admin
```
Comment (chrisnojima) [inline go/teams/service_helper.go]: Do we want this change?

Comment (chrisnojima) [reply go/teams/service_helper.go]: Description and avatar are the same?

**Comment by chrisnojima** [comment]
Cc: @mmaxim

File: go/teams/service_helper.go (hunk 1 above)
Comment (mmaxim) [reply go/teams/service_helper.go]: Definitely not.

**Comment by mmaxim** [comment]
If the avatar UI is using that field to control who can do it, we should just make a new field, not change the channel description one.

**Comment by chrisnojima** [comment]
That's what I thought thanks

File: go/teams/service_helper.go (hunk 1 above)
Comment (zanderz) [reply go/teams/service_helper.go]: so someone with write permission is supposed to be able to edit the team description? It didn't work that way yesterday, so I assumed they were equivalent.

Comment (mmaxim) [reply go/teams/service_helper.go]: Assuming this is used for how it is named, this is referring to channel description.

File: go/systests/teams_test.go [hunk 2]
```diff
@@ -1279,6 +1279,7 @@ func TestTeamCanUserPerform(t *testing.T) {
 	require.True(t, annPerms.DeleteChannel)
 	require.True(t, annPerms.RenameChannel)
 	require.True(t, annPerms.EditChannelDescription)
+	require.True(t, annPerms.ChangeChannelAvatar)
```
Comment (mmaxim) [inline go/systests/teams_test.go]: It's not a channel avatar, that is not a thing that exists. The avatar is for the team. I think the root cause here is that `EditChannelDescription` is used in the teams code for teams related stuff, beyond this avatar problem (which is I guess why you were involved with it in the first place). You should make a new field here called `EditTeamDescription`, and use it for both the avatars and the actual team description.

You get the same error trying to edit the team description as a writer, but no black bar for some reason. cc @buoyad

File: go/teams/service_helper.go (hunk 1 above)
Comment (zanderz) [reply go/teams/service_helper.go]: :white_check_mark:

File: go/systests/teams_test.go (hunk 2 above)
Comment (zanderz) [reply go/systests/teams_test.go]: :white_check_mark:

**Pull request merged**